### ISSUE TITLE
Support for styled components

### ISSUE THREAD

**Issue by skovhus** [ArnaudBarre/eslint-plugin-react-refresh#32]
Do we need some special treatment for styled components? At least I'm getting an unexpected error `Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components` for styled components like `AnotherReactContainer` in this example:

```tsx
import styled from "styled-components";

export const SomeReactComponent = () => (<div>Test</div>);

export const AnotherReactContainer = styled.div`
  padding-bottom: 6px;
`;
```

**Comment by ArnaudBarre** [comment]
Fixed in [0.4.5](https://github.com/ArnaudBarre/eslint-plugin-react-refresh/releases/tag/v0.4.5)!
